### PR TITLE
Remove unused UI elements from the Admin view.

### PR DESCRIPTION
**Pull request by EarthlingDavey**
Updates to remove unused UI elements. 

For all:
- Hide the colour and typography panels. 
- Removes support for custom class name on core blocks
- Remove the Openverse category on page editor.
- Filter the list of allowed block types in the block editor.
- Remove Relevancii from block editor header
- Remove custom taxonomies from document post type.
- Update the top left corner menu. Remove links to WP.
- Don't show document file type hint on Document upload.
  i.e. Are you uploading file types: doc, docx, pdf, xls, xlsx, zip etc. ? Try to add document instead.
- De-WordPress the redirect plugin's text.
- Redirect Plugin - force https by default. And 301 status by default.

For non-admins:
- Hide Relevanssi columns on the All Pages admin view.
- Hide the update and maintenance nags 
- Remove the User Searches & Admin Searches submenu
- Remove fields from update profile page
- Hide the Validate Structure sub-menu
- Hide the Custom Fields from the document edit page
- Hide the Custom Fields from the redirect edit page

### CODE REVIEW

File: public/app/themes/justice/jsconfig.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
         "jsx": "react-jsx"
-    }
+    },
+    "exclude": ["node_modules", "dist"]
 }
```
Comment (wilson1000) [inline public/app/themes/justice/jsconfig.json:6]: Might need a new line at the end

**Review by wilson1000 — DISMISSED** [review]
Looks good @EarthlingDavey 👍 
Just one silly comment, if you fancy updating it.

File: public/app/themes/justice/jsconfig.json (hunk 1 above)
Comment (EarthlingDavey) [reply public/app/themes/justice/jsconfig.json]: Cheers! This is sorted in the below commit.

File: public/app/themes/justice/inc/block-editor.php [hunk 2]
```diff
@@ -91,29 +91,37 @@ public function search(): string
...
     {
 
         // Match all a tags class of moj-anchor and a single space.
-        $pattern = '/(<a[^>]*class="moj-anchor"[^>]*>) (<\/a>)/';
+        $moj_anchor_pattern = '/(<a[^>]*class="moj-anchor"[^>]*>) (<\/a>)/';
         // Remove the single space.
-        $replacement = '$1$2';
-        $content = preg_replace($pattern, $replacement, $content);
+        $moj_anchor_replacement = '$1$2';
+        $content = preg_replace($moj_anchor_pattern, $moj_anchor_replacement, $content);
```
Comment (wilson1000) [inline public/app/themes/justice/inc/block-editor.php:115]: In the interest of saving bytes, could this just be the following? I understand if you're using comments to explain the rationale.

```php
$content = preg_replace($moj_anchor_pattern, '$1$2', $content);
```

**Review by wilson1000 — APPROVED** [review]

File: public/app/themes/justice/inc/block-editor.php (hunk 2 above)
Comment (EarthlingDavey) [reply public/app/themes/justice/inc/block-editor.php:115]: Yes, that's a good point!

**Review by wilson1000 — APPROVED** [review]
Looks fab 😄

**Pull request merged**